### PR TITLE
Navigation bar and CSS

### LINKED ISSUE #57: CSS style website
Improve look of the website

### PR DESCRIPTION
**Pull request by Dom123k**
- Added navigation bar to all pages
- Added CSS to all pages
- Error pages were excluded

closes #57

### CODE REVIEW

**Comment by thomas7carriere** [comment]
homepage
Is the GIF suppose to be there? lol

**Comment by Dom123k** [comment]
> homepage
> Is the GIF suppose to be there? lol

Hmmm yes, not sure why the gif is messed up on the screen, will try to have it positioned relative to the screen size.

This is how it looks on mine

Untitled

**Comment by thomas7carriere** [comment]
Seems to be the same for me in both Firefox and Chrome. Strange

**Review by MikeEvans6 — APPROVED** [review]
works fine for me on chrome

**Comment by thomas7carriere** [comment]
Hopefully just a problem on my end

**Review by thomas7carriere — APPROVED** [review]

File: src/main/resources/static/navBar.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title></title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <script type="text/javascript" src="https://code.jquery.com/jquery-latest.min.js"></script>
+</head>
+<style>
```
Comment (cyang030) [inline src/main/resources/static/navBar.html:8]: same as above

**Comment by cyang030** [comment]
CSS looks good for me at full screen, although the return monke gif will cover text if the vertical resolution is not high enough. (can be reproduced by resizing the window)
image

**Review by kellyharrison — APPROVED** [review]

File: src/main/resources/static/navBar.html (hunk 1 above)
Comment (Dom123k) [reply src/main/resources/static/navBar.html:8]: > CSS looks good for me at full screen, although the return monke gif will cover text if the vertical resolution is not high enough. (can be reproduced by resizing the window)
> image
@thomas7carriere 

I changed it so now the image should resize with the browser, would you able able to check that it works for you and the gif does not cover the navigation bar

Comment (Dom123k) [reply src/main/resources/static/navBar.html:8]: I'm not getting any warnings on Firefox, so I'm not sure.

I get this error "GET http://localhost:8080/favicon.ico" but I have no idea what that is about

Comment (thomas7carriere) [reply src/main/resources/static/navBar.html:8]: ya that fixed it. The favicon thing doesn't really matter, but you can specify one to make it go away.

**Pull request merged**